recipes/dcase2022_task4_audioset_baseline/train_sed.py

```diff
@@ -116,16 +116,14 @@ def single_run(
             pad_to=config["data"]["audio_max_len"],
         )
 
-        # audioset
         strong_df = pd.read_csv(config["data"]["strong_tsv"], sep="\t")
```

Comment:
There seems there is no option on YAML o via argparse to use the additional labels

Reply:
The baseline regards specifically Audioset and it is considered (at least so far) as a new baseline (even if the modifications made to the code are only really small). 
The audioset part of the set is concatenated to the synthetic set and treated the same. 
on the yaml file ([sed.yaml](https://github.com/DCASE-REPO/DESED_task/blob/ronchini_exp/recipes/dcase2022_task4_audioset_baseline/confs/sed.yaml)) for the Audioset baseline, the strong_folder has been added. 

But i do agree, a baseline with the option to add this could be considered as well, since the most of the code is the same.

Reply:
Ok, here right ? https://github.com/DCASE-REPO/DESED_task/blob/80a1521a78061c90a5268e19971d781c4c38100d/recipes/dcase2022_task4_audioset_baseline/train_sed.py#L164

Reply:
I would not do a different folder. I would use the same folder and scripts with an option via argparse for using the additional annotation. Otherwise it is a nightmare to maintain

Reply:
> Ok, here right ?
> 
> https://github.com/DCASE-REPO/DESED_task/blob/80a1521a78061c90a5268e19971d781c4c38100d/recipes/dcase2022_task4_audioset_baseline/train_sed.py#L164

Exactly

Reply:
> I would not do a different folder. I would use the same folder and scripts with an option via argparse for using the additional annotation. Otherwise it is a nightmare to maintain

I agree on that